conf/modules.config

```diff
@@ -151,12 +151,21 @@ process {
         ]
     }
 
-    withName: TRANSDECODER {
+    withName: LONGORF {
```

Comment:
Do we really need to publish files both for `longorf` and for `predict`? I think it would be tidier if we just kept the output form the process that actually produces the orf's we want to continue processing.

Reply:
so we can keep `predict`, right? I wasn't sure so I decided to put both :)

Reply:
If `predict` is the one to produce the `gff` and `faa` and other files we want for downstream, then yes, let's keep that one!

Reply:
It is! I will push the update in few minutes!

Reply:
Done!